src/font.ts

```diff
@@ -7,6 +7,15 @@ import { PetalumaMetrics } from './fonts/petaluma_metrics';
 import { CustomFont } from './fonts/custom_glyphs';
 import { CustomMetrics } from './fonts/custom_metrics';
 
+export interface FontStruct {
```

Comment:
This is related to text fonts, so prob not the right place (which is music glyphs.) It's a good struct though, maybe better in `common`, also you can call it `FontInfo`.

Reply:
Done